### ISSUE TITLE
ansible-lint seems to ignore the `args:` method to define arguments

### ISSUE THREAD

**Issue by jbruggem** [willthames/ansible-lint#103]
ansible-lint seems to ignore the `args:` method to define arguments

For example I get this error:
```
[ANSIBLE0004] Git checkouts must contain explicit version
```

With this code:
```yaml
- name: checkout repo
  git:
  args:
    repo: "{{ repo_path }}"
    dest: "{{ checkout_folder }}"
    version: "{{ repo_version }}"
```

But not with this code: 
```yaml
- name: checkout repo
  git: version="{{ repo_version }}"
  args:
    repo: "{{ repo_path }}"
    dest: "{{ checkout_folder }}"
```

**Comment by willthames** [comment]
If you're using Ansible 2.0, can you check if it works with PR #91. If you're using Ansible 1.9, I can't see this being fixed without a PR.

I concede that this is a bug, but I don't understand the use case - why not use
```
- name: checkout repo
  git:
    version: "{{repo_version}}"
    repo: "{{repo_path}}"
    dest: "{{checkout_folder}}"
```
if you don't want the single line style.

**Comment by willthames** [comment]
This fix won't make it in with 2.5, but I'll look at fixing this next.

**Comment by jbruggem** [comment]
great! thanks :)
